src/bin/lpython.cpp

```diff
@@ -1657,6 +1657,8 @@ int main(int argc, char *argv[])
                 if (err != 0) {
                     if (0 < err && err < 256) {
                         return err;
+                    } else if (err % 256 == 0) {
+                        return err / 256;
```

Comment:
What does this do exactly? Let's document it in a comment.

Reply:
In Linux, the exit values are multiplied by 256
https://stackoverflow.com/a/3736358
unix.stackexchange.com/questions/394639/why-do-high-exit-codes-on-linux-shells-256-not-work-as-expected
So, when we do:
```py
quit(10)
```
```
>> echo $?
2560
```